dynamodb/src/main/scala/akka/stream/alpakka/dynamodb/scaladsl/DynamoDb.scala

```diff
@@ -27,6 +28,38 @@ object DynamoDb {
   )(implicit client: DynamoDbAsyncClient, operation: DynamoDbOp[In, Out]): Flow[In, Out, NotUsed] =
     Flow[In].mapAsync(parallelism)(operation.execute(_))
 
+  /**
+   * Create a Flow that emits a response for every request to DynamoDB.
+   * A successful response is wrapped in [scala.util.success] and a failed
+   * response is wrapped in [scala.util.Failure].
+   */
```

Comment:
Add `@param` documentation with an explanation of `parallelism`.
Add `@tparam` for `State`.